estate/models/estate_property.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real estate property"
+
+    name = fields.Char("Property Name", required=True)
+    description = fields.Text("Property description")
+    active = fields.Boolean("Active", default=True)
```

Comment:
By default, the label will take the name of the field and capitalize it. So in this case it will be, by default, `Active`. So we don't need to manually add it.